### ISSUE TITLE
CTRL+W does not work with 9.0 RC

### ISSUE THREAD

**Issue by PoloniumRain** [d2phap/ImageGlass#1667]
### System information:
 - Windows OS version: 10 Pro
 - ImageGlass version: 9.0 RC with .NET 8

### To Reproduce
1. Open IG
2. Press CTRL+W keys
3. IG will not close

IG 9.0 RC no longer closes when pressing the CTRL+W keys. This works in previous releases. I'm assuming this is a bug or just a feature that is not implement yet?

And as some general feedback, overall i'm _really_ liking 9.0 RC. It's a clear improvement over all previous versions. It fixes a LOT of the main issues i've had with IG for many years, especially when it comes to performance. Some areas where the Windows Photos app had a very obvious performance advantage over IG were with things like loading very high res images, certain file types, zooming in/out, going to the next/previous image, and panning/moving images (especially when using bicubic interpolation). Basic everyday tasks yet IG was always far behind. With IG 9.0 the gap has been greatly reduced and in some cases it's just as fast. Great work :)

**Comment by d2phap** [comment]
Hotkeys are reworked in v9 for customization support.
You can edit the `igconfig.json` to add Ctrl+W hotkey for closing IG:

```ts
// in igconfig.json
{
  "MenuHotkeys": {
    "MnuExit": ["Escape", "Ctrl+W"]
  }
}
```

No setting UI for this yet because. Make sure you close IG before editing the `igconfig.json`.

**Comment by PoloniumRain** [comment]
Thanks that works well. 

Will this eventually be the default setting? Because even if it will be customisable i still think that CTRL+W should be default because it's so commonly used. I even have one of my mouse buttons mapped to CTRL+W.

**Comment by d2phap** [comment]
I will add it together with ESC as default hotkeys to close the app

**Comment by d2phap** [comment]
Fixed in [ImageGlass 9.0.7.1125](https://github.com/d2phap/ImageGlass/releases/tag/9.0.7.1125)
